tests/test_auto_detection.py

```diff
@@ -27,10 +27,6 @@ async def app(scope, receive, send):
     pass  # pragma: no cover
 
 
-# TODO: Add pypy to our testing matrix, and assert we get the correct classes
-#       dependent on the platform we're running the tests under.
```

Comment:
@tomchristie Do you remember what do you mean by "assert we get the correct classes dependent..." ?